src/App.js

```diff
@@ -0,0 +1,23 @@
+import { Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Line from './components/Line';
+import Rockets from './components/Rockets';
+import MyProfile from './components/MyProfile';
+import Missions from './components/Missions';
+
+function App() {
+  return (
+    <>
+      <Navbar />
+      <Line />
+      <Routes>
+        <Route path="/rockets" element={<Rockets />} />
+        <Route path="/missions" element={<Missions />} />
+        <Route path="/my-profile" element={<MyProfile />} />
+        <Route path="/" element={<Rockets />} />
```

Comment:
[OPTIONAL] I think there is no need to repeat the rockets route here, it's fine if you leave it to be just the homepage, that way we avoid repetition.